src/cf.data.pre

```diff
@@ -4413,6 +4413,41 @@ DOC_START
 		cache_dir rock /ssd3 ... max-size=99999
 DOC_END
 
+NAME: paranoid_hit_validation
+COMMENT: time-units-small
+TYPE: time_nanoseconds
+DEFAULT: 0
+DEFAULT_DOC: validation disabled
+LOC: Config.paranoid_hit_validation
+DOC_START
+		Controls whether Squid should perform paranoid validation of cache entry
+		metadata integrity every time a cache entry is hit. Squid bugs
+		notwithstanding, this low-level validation should always succeed. Each
+		failed validation results in a cache miss, a BUG line reported to cache.log,
+		and the invalid entry marked as unusable (and eventually purged from the
+		cache).
+
+		Squid can only validate shared cache memory and rock cache_dir entries.
+
+		Positive paranoid_hit_validation values enable validation and approximate
```

Comment:
Please make this part a  bullet-point list rather than a hugs block of text. Understanding these details are critical to using the directive properly.

No description of what happens when the value is negative.

Reply:
... or the special 0 case.